layers/+lang/javascript/packages.el

```diff
@@ -62,13 +62,13 @@
   (spacemacs/set-leader-keys-for-major-mode 'js2-mode
     "I" 'spacemacs/impatient-mode))
 
-(defun javascript/init-import-js ()
-  (use-package import-js
-    :defer t
-    :init
-    (progn
-      (add-hook 'js2-mode-hook #'run-import-js)
-      (spacemacs/import-js-set-key-bindings 'js2-mode))))
+;; (defun javascript/init-import-js ()
```

Comment:
I think this block should rather be removed then be kept as comment.
If it is kept as comment you should add a comment refering to your change and explaining shortly why this is deactivated.

Reply:
Thanks, I'll delete that part and add change log